src/ffi.rs

```diff
@@ -33,6 +33,7 @@ pub enum DartTypedDataType {
     Float64 = 11,
     Float32x4 = 12,
     Invalid = 13,
+    Usize = 14,
```

Comment:
Is the `Usize` variant is implemented in `dart_api.h`? sounds like I can't find it anywhere [here](https://github.com/dart-lang/sdk/blob/4d77e3e64554f260a60f6cbca158a50f6a63b2a2/runtime/include/dart_api.h#L2491-L2508) that means it is not a thing internally in the dartvm.